packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -106,7 +108,13 @@ async function initProject(appName: string, options: CLIOptions) {
 
     console.log(`Creating a Contentful app in ${highlight(tildify(fullAppFolder))}.`);
 
-    await cloneTemplateIn(fullAppFolder, normalizedOptions);
+    const isInteractive = !normalizedOptions.example && !normalizedOptions.source && !normalizedOptions.javascript && !normalizedOptions.typescript
+
+    const templateSource = await getTemplateSource(options)
+
+    track({ template: templateSource, manager: normalizedOptions.npm ? 'npm' : 'yarn', interactive: isInteractive})
```

Comment:
Do we want to also track app's (folder) name?

Reply:
To keep the data as anonymous as possible, I would not do that.